includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```diff
@@ -1004,6 +1004,40 @@ public function generate_upe_checkout_experience_accepted_payments_html( $key, $
 		return $this->generate_title_html( $key, $data );
 	}
 
+	private function get_payment_method_description( $method ) {
+		$description_mapper = [
+			'card' => __(
+				'Let your customers pay with major credit and debit cards without leaving your store.',
+				'woocommerce-gateway-stripe'
+			),
+			'giropay' => __(
+				'Expand your business with giropay — Germany’s second most popular payment system.',
```

Comment:
Design and [this file](https://github.com/woocommerce/woocommerce-gateway-stripe/blob/develop/client/payment-methods-map.js) both has the string as `giropay`. But should I make the first letter cap here? (ex: `Giropay`)

Reply:
`giropay` is one of the few payment methods that are lowercase: https://stripe.com/docs/payments/giropay